packages/common/src/controllers/CheckoutController.ts

```diff
@@ -46,15 +46,17 @@ export default class CheckoutController {
     return this.accountService.svodOfferIds;
   };
 
-  createOrder = async (offer: Offer, paymentMethodId?: number): Promise<void> => {
+  createOrder = async (offer: Offer): Promise<void> => {
     const { getAccountInfo } = useAccountStore.getState();
     const { customer } = getAccountInfo();
 
+    const paymentMethods = await this.getPaymentMethods();
+    const paymentMethodId = paymentMethods[0]?.id;
```

Comment:
Won't it be a problem in case several methods are available?

Reply:
The logic hasn't changed and sets the initial payment method ID when initializing the order for the checkout form. When the user changes the payment method, the `updateOrder` method is called with the correct payment method id.